docs/stdlib/datetime.rst

```diff
@@ -662,6 +666,10 @@ EdgeDB stores and outputs timezone-aware values in UTC.
 
     Return the date and time of the start of the current transaction.
 
+    This function is non-volatile since it returns the current time when the
+    transaction is started, not when the function is called. As a result, it
+    can be used in :ref:`computed properties <ref_datamodel_computed>`, even
+    when they are defined in schema.
```

Comment:
A bit of a nit. Maybe drop `, even when they are` since it implies this has something to do with computeds outside of schema?

Reply:
My fear for leaving out any mention of definition in schema is that it might imply there are cases when a volatile function cannot be used in a computed property that is *not* inside schema.

> As a result, it can be used in computed properties.

Maybe if I reword like this?

> As a result, it can be used in schema-defined computed properties.

Reply:
I was thinking it would read 
> As a result, it can be used in computed properties defined in schema.